ci/TriggerRedHatBuild.groovy

```diff
@@ -0,0 +1,192 @@
+/*
```

Comment:
Copied from docker-nexus3 repo. When I find a 3rd use, then we can roll this into a shared pipeline or something.